lib/workers/worker.ts

```diff
@@ -23,25 +23,29 @@ import { SqsEventSource } from "@aws-cdk/aws-lambda-event-sources";
 
 export class Worker extends Construct {
 
+    private _function: lambda.Function
+
     constructor(scope: Construct, id: string, props: WorkerProps) {
         super(scope, id);
         this.createConstructs(id, props);
     }
 
     private createConstructs(id: string, props: WorkerProps) {
         const extraSecurityGroups = this.node.tryGetContext("extraIngressSecurityGroups");
+        const accumuloPassword = Math.random().toString(36).substr(2, 8);
 
         // Build environment for Lambda
         const environment: { [id: string] : string; } = {
             "cluster_name": props.cluster.clusterName,
-            "graph_table_name": props.graphTable.tableName
+            "graph_table_name": props.graphTable.tableName,
+            "accumuloPassword": accumuloPassword
```

Comment:
Not sure we should have the same password for every user (gaffer, tracer, root etc) on every graph. Does this also result in the password being placed in plaintext in the cloudformation template?

Reply:
This was a hack to work around an authentication error from cycling round adding and uninstalling a graph while testing, you're right the password is in plain text in the template.

Reply:
Ah okay fair enough. so is it possible to remove this?

Reply:
I've reverted this back.